assessments/projects/tikv/self-assessment.md

```diff
@@ -0,0 +1,335 @@
+* **Maintainers**: Responsible for moderating the discussions, managing project release, and proposing new committers or maintainers
+
+In addition, there is a separate infrastructure team, not responsible for the team decision-making process, but that helps on applying the decision. The current list of infrastructure members is provided with their emails and githubs on the TiKV Governance page.
+* **Infrastructure team**: Responsible for setting administration, security, and supporting the existing teams
+
+**Voting**
+* Decisions regarding the project are made by votes on the primary project community repository.
+* Votes are indicated via pull requests under the votes folder.
+* Positive votes require no explanation, while vetoes (negative votes) must have an explanation. 
+* Required vote counts can vary based on the decision but typically require at least 2 positive votes.
+
+* Internal Communication: Slack 
+
+ * Inbound Communication: Twitter, Blog, Reddit, Stack Overflow
+
+* Outbound Communication: None
+
+**Ecosystem**
+* At the time of writing, there are currently four client drivers: Go, Java, Rust, and C
+  - The Go client is the only stable client
+  - The Java and Rust clients are considered unstable
+  - The C client is currently in early development
+
+TiKV provides two separate clients: Raw and Transactional
+* ***Raw***: A lower-level key-value API for interacting directly with individual key-value pairs
+* ***Transactional***: A higher level key value API that provides ACID semantics
+
+## Security issue resolution
+
+***Note:*** When looking for methods of security issue resolution, there was no easily accessible area to discuss each protocol. This could potentially lead to mishaps like major security vulnerabilities being publicly shown as issues
+
+* Bug Fixes or documentation improvement can be implemented via a normal Github Pull Request workload
+* To contribute “substantial changes”, a RFC (request for comments) document provides a consistent and controlled path for new features to enter the project
+  - RFCs must provide summary, motivation, detailed design, drawbacks, alternative, and unanswered questions
+* To report a vital security vulnerability, an email should be sent to the TiKV-security group (tikv-security@lists.cncf.io), and encrypted using the public PGP key
+  - Disclosure Policy:
+    - The received security vulnerability report shall be handed over to the security team for follow-up coordination and repair work.
+    - After the vulnerability is confirmed, a draft Security Advisory will be created on Github that lists the details of the vulnerability.
+    - Invite related personnel to discuss the fix.
+    - Fork the temporary private repository on Github, and collaborate to fix the vulnerability.
+    - After the fixed code is merged into all supported versions, the vulnerability will be publicly posted in the GitHub Advisory Database.
+
+## Appendix
+
+***Known Issues Over Time***
```

Comment:
Most of these seem like security audit or disclosure from the team themselves, thus, outside the scope of security pals efforts. I would suggest providing a short summary with references to find details.

Reply:
We have shortened this section and added relevant links for further exploration of this topic if the reader is interested.